src/frontend/src/pages/AdminToolsPage/RecruitmentConfig/FAQTable.tsx

```diff
@@ -96,6 +118,19 @@ const FAQsTable = () => {
           Add FAQ
         </NERButton>
       </Box>
+      {faqToDelete && (
+        <NERDeleteModal
+          open={faqToDelete !== undefined}
+          onHide={() => setFaqToDelete(undefined)}
+          formId="delete-item-form"
+          title="FAQ"
+          onFormSubmit={() => {
+            if (faqToDelete) {
```

Comment:
You can just do `handleDelete(faqToDelete.faqId`

Reply:
If i remove the conditional, it will say faqToDelete isnt undefined